### PR TITLE
Fix issue caused by Quota threshold inconsistency

### PR DESCRIPTION
**Pull request by P-Cao**
# Description
Now not setting threshold percent data as response since PAPI might return inconsistent result due to accuracy loss
Percent threshold can be calculated by actual threshold
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by P-Cao** [comment]
run e2e test Quota

**Comment by P-Cao** [comment]
run e2e test Quota

**Comment by P-Cao** [comment]
UT:
image
FT:
image

**Comment by P-Cao** [comment]
run e2e test Quota

**Review by baoy1 — APPROVED** [review]

File: powerscale/provider/quota_resource.go [hunk 1]
```diff
@@ -319,16 +319,14 @@ func (r *QuotaResource) Schema(ctx context.Context, req resource.SchemaRequest,
 						Computed:            true,
 					},
 					"percent_advisory": schema.NumberAttribute{
-						Description:         "Advisory threshold as percent of hard threshold. Usage bytes at which notifications will be sent but writes will not be denied. Must be >= 0.01 <= 99.99, precision 2",
```
Comment (Krunal-Thakkar) [inline powerscale/provider/quota_resource.go:322]: I think Description should be as it is.

**Review by Krunal-Thakkar — APPROVED** [review]

**Pull request merged**